frontend/modelarchive/src/test/java/org/pytorch/serve/archive/ModelArchiveTest.java

```diff
@@ -91,4 +91,10 @@ public void testMalformURL() throws ModelException, IOException, InterruptedExce
         ModelArchive.downloadModel(
                 modelStore, "https://../model-server/models/squeezenet_v1.1/squeezenet_v1.1.mod");
     }
+
+    @Test(expectedExceptions = DownloadModelException.class)
+    public void testMalformLocalURL() throws ModelException, IOException, InterruptedException {
+        String modelStore = "src/test/resources/models";
+        ModelArchive.downloadModel(modelStore, "file://" + modelStore + "/mnist1.mar");
```

Comment:
This test case only tests the "mar file already exists" scenario. Please add test cases for the following use cases:

 - Valid local mar file URL, validate that the mar file correctly gets copied to the model-store.
 - URL path does not exist
 -